front/temporal/labs/utils/modjo.ts

```diff
@@ -0,0 +1,380 @@
+import { isModjoCredentials, OAuthAPI } from "@dust-tt/types";
+import { either } from "fp-ts";
+import { pipe } from "fp-ts/function";
+import * as t from "io-ts";
+
+import config from "@app/lib/api/config";
+import type { Authenticator } from "@app/lib/auth";
+import { getFeatureFlags } from "@app/lib/auth";
+import type { LabsTranscriptsConfigurationResource } from "@app/lib/resources/labs_transcripts_resource";
+import type { Logger } from "@app/logger/logger";
+import logger from "@app/logger/logger";
+
+const ModjoSpeaker = t.type({
```

Comment:
Same for the naming here, the schemas need to be suffixed with Schema otherwise this looks like a type